content/en/opentelemetry/interoperability/environment_variable_support.md

```diff
@@ -0,0 +1,155 @@
+Mapped values between `OTEL_TRACES_SAMPLER` & `DD_TRACE_SAMPLE_RATE`:<br>
+  - `parentbased_always_on`|`1.0`
+  - `parentbased_always_off`|`0.0`
+  - `parentbased_traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+  - `always_on`|`1.0`
+  - `always_off`|`0.0`
+  - `traceidratio`|`${OTEL_TRACES_SAMPLER_ARG}`
+
+`OTEL_TRACES_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_ENABLED=false` <br>
+Trace exporter to be used<br>
+**Notes**: only a value of `none ` is accepted<br>
+
+`OTEL_METRICS_EXPORTER`
+: **Corresponding Datadog Environment Variable**: `DD_RUNTIME_METRICS_ENABLED=false` <br>
+Metrics exporter to be used<br>
+**Notes**: only a value of `none` is accepted<br>
+**Not Supported In**: `php` sdk<br>
+
+`OTEL_RESOURCE_ATTRIBUTES`
+: **Corresponding Datadog Environment Variable**: `DD_TAGS` <br>
+Key-value pairs to be used as resource attributes. See [Resource semantic conventions][11] for details<br>
+**Notes**: `deployment.environment` maps to the `DD_ENV` environment variable<br>
+`service.name` maps to the `DD_SERVICE` environment variable<br>
+`service.version` maps to the `DD_VERSION` environment variable
+
+`OTEL_SDK_DISABLED`
+: **Corresponding Datadog Environment Variable**: `!DD_TRACE_OTEL_ENABLED` <br>
+Disable the SDK for all signals<br>
+**Notes**: Mapped values between `OTEL_SDK_DISABLED` & `DD_TRACE_OTEL_ENABLED`:<br>
+  - `true`|`false`
+  - `false`|`true`
+**Not Supported In**: `ruby` & `golang` sdk's<br>
+
+[for more info, see the OpenTelemetry documentation][10]
+## Java Specific Configuration
+
+`OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`
+: **Corresponding Datadog Environment Variable**: `!DD_INTEGRATIONS_ENABLED` <br>
+Set to `false` to disable all instrumentation in the agent<br>
+**Notes**: Mapped values between `OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED` & `DD_INTEGRATIONS_ENABLED`:<br>
+  - `true`|`false`
+  - `false`|`true`
+
+`OTEL_INSTRUMENTATION_[NAME]_ENABLED`
+: **Description**: Enables/disables the named OTel drop-in instrumentation<br>
+
+`OTEL_JAVAAGENT_CONFIGURATION_FILE`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_CONFIG` <br>
+Path to valid Java properties file which contains the agent configuration<br>
+
+`OTEL_INSTRUMENTATION_HTTP_CLIENT_CAPTURE_REQUEST_HEADERS`
+: **Corresponding Datadog Environment Variable**: `DD_TRACE_REQUEST_HEADER_TAGS` <br>
+A comma-separated list of HTTP header names. HTTP client instrumentations will capture HTTP request header values for all configured header names<br>
+**Notes**: Note there is a difference in the generated request tag name:<br>
+  - Datadog:	 `http.request.headers.<header-name>`
+  - OTel:		   `http.request.header.<header-name>`
```

Comment:
To clarify, when we map the OTel setting we follow the OTel naming convention for the generated tag name.

We might want to call this out more explicitly:
```suggestion
**Notes**: Header tagging configured using OTel environment variables follows the OTel tag name convention of `http.request.header.<header-name>` rather than the Datadog convention of `http.request.headers.<header-name>`<br>
```